packages/ui/src/app/pages/Validators/ValidatorList.stories.tsx

```diff
@@ -113,4 +143,78 @@ export default {
 
 type Story = StoryObj<typeof ValidatorList>
 
-export const Statistics: Story = {}
+export const StatisticsAndLists: Story = {}
+
+export const TestsFilters: Story = {
+  play: async ({ canvasElement, step }) => {
+    const screen = within(canvasElement)
+
+    const searchElement = screen.getByPlaceholderText('Search')
+    const verificationFilter = screen.getAllByText('Verification')[0]
+    const stateFilter = screen.getAllByText('State')[0]
+    const clearFilter = async () => {
+      expect(screen.getByText('Clear all filters'))
+      await userEvent.click(screen.getByText('Clear all filters'))
+    }
+    const expectNumberOfValidatorsToBe = async (n: number) => {
+      const liElements = screen.queryAllByRole('listitem')
+      const validatorElements = liElements.filter((li) => li.textContent?.includes('Nominate'))
+      await expect(validatorElements.length).toBe(n)
+    }
+
+    await step('Verifcation Filter', async () => {
+      await selectFromDropdown(screen, verificationFilter, 'verified')
+      await expect(screen.queryByText('unverifed')).toBeNull()
```

Comment:
The `queryBy...` methods are synchronous too. Only `findBy...` are async.
```suggestion
      expect(screen.queryByText('unverifed')).toBeNull()
```